src/components/molecules/card/card.scss

```
@@ -0,0 +1,50 @@
+@use "../../variables" as *;
+.card {
+  .card-img-top {
+    position: relative;
+
+    &:hover {
+      overflow: hidden;
+      transform: scale(1.1);
+      transition: 0.2s ease;
+    }
+  }
+
+  .card-img {
+    overflow: hidden;
+  }
+
+  .card-body {
+    padding: initial;
+  }
+
+  .card-imgframe {
+    z-index: 9;
+    position: absolute;
+    top: 0;
+    border-bottom: 150px solid transparent;
+    border-left: 150px solid $white;
+  }
+
+  .title {
+    position: absolute;
+    margin: 0px 6px;
+    z-index: 10;
+    top: 0px;
+    left: 6px;
+    color: $orange_bistre;
+    font-size: $fontsize-l;
+  }
+
+  .text {
+    margin: 32px 30px 32px;
+    font-size: $fontsize-s;
+  }
+
+  .card-button {
+    display: flex;
+    justify-content: center;
+    margin: 0px 32px 37px;
+    line-height: 33.4px;
+  }

```

```suggestion
  .card-body {
    padding: initial;
    padding: 32px;
  }

  .card-imgframe {
    z-index: 9;
    position: absolute;
    top: 0;
    border-bottom: 150px solid transparent;
    border-left: 150px solid $white;
  }

  .title {
    position: absolute;
    margin: 0px 6px;
    z-index: 10;
    top: 0px;
    left: 6px;
    color: $orange_bistre;
    font-size: $fontsize-l;
  }

  .text {
    font-size: $fontsize-s;
  }

  .card-button {
    display: flex;
    justify-content: center;
    line-height: 33.4px;
  }
```
現状でも全く問題ありませんが、今回のような余白の場合、子要素にmarginを設定するのではなく親要素( `card-body` )にpaddingを設定したほうがスッキリ書けるのでおすすめです。